docs/Node Operators/configuration-variables.md

```diff
@@ -363,8 +384,6 @@ Because of the complications with advisory locks, Chainlink nodes with v1.1.0 an
 - Node B spinlocks and checks periodically to see if the client ID is too old. If the client ID is not updated after a period of time, node B assumes that node A failed and takes over. Node B becomes the owner of the row and updates the client ID once per second.
 - If node A comes back, it attempts to take out a lease, realizes that the database has been leased to another process, and exits the entire application immediately.
 
-The default is set to `advisorylock`.
```

Comment:
The default is set to `dual` (for backwards compatibility).

Reply:
The whole section above this should probably be reworked though because of the new "dual" default:
> By default, Chainlink nodes use an advisory lock to manage this. However, advisory locks come with several problems:

Reply:
I decided to keep most of this information, but adjusted some of the text to more accurately explain the defaults. The details comparing lease and advisory locks are still important.